WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewerViewModel.cs

```diff
@@ -24,17 +24,33 @@ namespace WalletWasabi.Gui.Controls.WalletExplorer
 {
 	public class TransactionViewerViewModel : WasabiDocumentTabViewModel
 	{
+		private readonly int _jsonCharLimit = 2_500;
 		private string _txId;
-		private string _psbtJsonText;
+		private string _psbtJsonText; 
+		private string _truncatedPsbtJsonText;
 		private string _psbtHexText;
 		private string _psbtBase64Text;
 		private byte[] _psbtBytes;
+		private bool _IsPsbtJsonTextTruncated;
 
 		public TransactionViewerViewModel() : base("Transaction")
 		{
 			Global = Locator.Current.GetService<Global>();
 
 			OpenTransactionBroadcaster = ReactiveCommand.Create(() => IoC.Get<IShell>().AddOrSelectDocument(() => new TransactionBroadcasterViewModel()));
+
+			CopyBase64Psbt = ReactiveCommand.CreateFromTask(async () =>
+			{
```

Comment:
Idk if this is too simple; perhaps I should add a try/catch here too?

Reply:
We are using this pattern for that: 
https://github.com/molnard/WalletWasabi/blob/a79fc88beb7d5d32f535b81f0a9791b065033495/WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs#L122

Reply:
ah yes ofcourse haha :) my bad